src/modules/OTA-api/data-components/Contributors.tsx

```diff
@@ -59,20 +84,25 @@ const Contributors: React.FC<ContributorsProps> = React.memo(
       >
         {subtitle && <h4 className={classNames(s.contributors_subtitle)}>{subtitle}</h4>}
         <div className={s.contributors_items}>
-          {Object.entries(contributors).map(([name, url]) => {
+          {Object.entries(contributors).map(([name, value]) => {
+            const website = typeof value === 'object' ? value.website : value;
+            const imgSrc =
+              typeof value === 'object' && value.picture != null
+                ? value.picture
+                : `/images/contributors/${slugify(name, { lower: true })}.jpg`;
```

Comment:
I don't understand why we add all this logic. Why keep the inference based on `slugify` when the vast majority of use cases will be to have a full `avatar_url`? Wouldn't it be much simpler to unify the `staff` data schema with the All Contributors one and simply adapt the data loading from it? 🙂

Reply:
I've thought about it, it's simpler and easier to read but I haven't developed the data model because we haven't talked about it and I've already spent a lot of time implementing something that works.

Reply:
We agreed synchronously today to use `staff.json` only to list IDs of data in allcontributorsrc 🙂